app/0.14.x/pdk/index.md

```diff
@@ -47,7 +47,7 @@ The Plugin Development Kit (or "PDK") is set of Lua functions and variables
 
 
 
-### <a name="kong_version"></a>kong.version
+#### <a name="kong_version"></a>kong.version
```

Comment:
Please remove the `<a name="kong_version"></a>` code from headers - it is not needed